### PR TITLE
libvirt_memory: install numactl in vm

### PR DESCRIPTION
**Pull request by dzhengfy**
Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

**Comment by dzhengfy** [comment]
```
2024-04-11 05:23:58,059 output           L0702 DEBUG| [stderr] DEBUG:aexpect.client:[192.168.122.6] Sending command: rpm -q numactl

2024-04-11 05:23:58,200 output           L0702 DEBUG| [stderr] DEBUG:aexpect.client:[192.168.122.6] Sending command: echo $?

2024-04-11 05:23:58,301 output           L0702 DEBUG| [stderr] DEBUG:aexpect.client:[192.168.122.6] Sending command: yum install -y numactl

2024-04-11 05:24:08,580 output           L0702 DEBUG| [stderr] DEBUG:aexpect.client:[192.168.122.6] Sending command: echo $?

2024-04-11 05:24:10,583 output           L0702 DEBUG| [stderr] DEBUG:aexpect.client:[192.168.122.6] Sending command: memhog 1083424k

```
libvirt-10.0.0-6.1
 (1/1) type_specific.io-github-autotest-libvirt.guest_numa_node_tuning.memory_binding_with_emulator_thread.default_pagesize.mem_mode_strict.nodeset_no_overlap_with_emulatorpin.emulatorpin_single_node: PASS (72.40 s)

**Review by cliping — APPROVED** [review]
LGTM

File: virttest/utils_libvirt/libvirt_memory.py [hunk 1]
```diff
@@ -79,6 +80,8 @@ def consume_vm_freememory(vm_session, consume_value=100000):
     """
     vm_session.cmd_status("swapoff -a")
     free_mem = utils_memory.freememtotal(vm_session)
+    if not utils_package.package_install("numactl", vm_session):
+        raise exceptions.TestError("Fail to install memhog")
```
Comment (chloerh) [inline virttest/utils_libvirt/libvirt_memory.py:84]: Seems we could make the error message a little bit more clearer by adding `numactl` into it. Something like 
`"Fail to install pkg numactl which provide command 'memhog'"`

**Review by chloerh — COMMENTED** [review]
One small suggestion.

File: virttest/utils_libvirt/libvirt_memory.py (hunk 1 above)
Comment (dzhengfy) [reply virttest/utils_libvirt/libvirt_memory.py]: Updated. thanks

**Review by chloerh — APPROVED** [review]

**Pull request merged**